QUEST_LV_0200.tsv

```diff
@@ -2597,8 +2597,8 @@ QUEST_LV_0200_20150317_002596	There is a suspicious shadow in Spring Light Woods
 QUEST_LV_0200_20150317_002597	Chafer was hiding in the altar. Defeat the chafer.
 QUEST_LV_0200_20150317_002598	Check the destroyed seal tower
 QUEST_LV_0200_20150317_002599	Someone destroyed the tower. Check its status.
-QUEST_LV_0200_20150317_002600	Defeat the Mantiken
-QUEST_LV_0200_20150317_002601	Seems like the Mantiken destroyed the tower. Defeat it.
+QUEST_LV_0200_20150317_002600	$Defeat the Mentiken
+QUEST_LV_0200_20150317_002601	$Seems like the Mentiken destroyed the tower. Defeat it.
```

Comment:
Seems like the Mentiken destroyed the tower. -> Seems like Manticen destroyed the tower.